test/test_automl/test_automl.py

```diff
@@ -426,6 +427,16 @@ def test_do_dummy_prediction(backend, dask_client, datasets):
         'predictions_ensemble_1_1_0.0.npy')
     )
 
+    model_path = os.path.join(backend.temporary_directory, '.auto-sklearn',
```

Comment:
While I think we should test for the dummy models complying with sklearn, we should not do so here. Could you please create a test in the evaluation module tests?